_docs/tasks/metrics-logs.md

```diff
@@ -9,210 +10,300 @@ type: markdown
 ---
 {% include home.html %}
 
-This task shows how to configure Mixer to automatically gather telemetry
-for a service within a cluster. At the end of this task, a new metric and
-a new log stream will be enabled for calls to a specific service within your
-cluster.
+This task shows how to configure Istio to automatically gather telemetry for
+services in a mesh. At the end of this task, a new metric and a new log stream
+will be enabled for calls to services within your mesh.
 
 The [BookInfo]({{home}}/docs/samples/bookinfo.html) sample application is used
 as the example application throughout this task.
 
 ## Before you begin
-* [Install Istio](./installing-istio.html) in your kubernetes
-  cluster and deploy an application.
-
-* Configure your environment to support calling `istioctl mixer`.
-  This may require setting up port-forwarding for the Mixer Config API as described in the
-  [reference docs]({{home}}/docs/reference/commands/istioctl.html#istioctl-mixer) for `istioctl mixer`.
+* [Install Istio](./installing-istio.html) in your kubernetes cluster and deploy
+  an application. This task assumes that Mixer is setup in a default
+  configuration (`--configDefaultNamespace=istio-config-default`). If a
```

Comment:
do we have a namespace for istio config? isn't it istio-system ?

Reply:
Mixer, by default, uses `istio-config-default`. It is configurable, but using `istio-system` is potentially problematic, iirc (issue: how to specify configuration just for the istio components themselves, but not affect global configuration).